lua/gruvbox_crisp/init.lua

```
@@ -0,0 +1,75 @@
+
+  local specs = groups.get(p, o)
+  for name, spec in pairs(specs) do
+    if spec.link then
+      vim.api.nvim_set_hl(0, name, { link = spec.link, default = false })
+    else
+      vim.api.nvim_set_hl(0, name, spec)
+    end
+  end
+
+  -- Ensure whitespace and special keys are visible but subdued
+  vim.api.nvim_set_hl(0, "Whitespace", { fg = p.border_neutral })
```

Luacheck (1.2.0)

[warning] accessing undefined variable 'vim'

(W113)